runtime/libpgmath/test/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ if(${LIBPGMATH_STANDALONE_BUILD})
   endif()
   set(LIBPGMATH_LIT_ARGS "${LIBPGMATH_LIT_ARGS_DEFAULT}" CACHE STRING
     "Default options for lit")
-  separate_arguments(LIBPGMATH_LIT_ARGS)
+  separate_arguments(LIBPGMATH_LIT_ARGS NATIVE_COMMAND)
```

Comment:
This needs CMake 3.9. Can you increase the minimum requirement where it is appropriate?